_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "From the editor's desk: OpenSearch and inclusion"
+authors: 
+  - nbower
+date: 2022-08-31
+categories: 
+  - community
+---
+
+As part of our commitment to providing complete and best-in-class documentation to the OpenSearch community, we want to ensure that you have visibility into how we approach creating bar-raising content. To that end, we will be publishing regular blog posts with tips, guidance, and best practices that will help contributors collaborate with us. Subjects may include technical writing, style, voice and tone, brand messaging, and other topics relevant to the production of our documentation. We hope you find this to be useful, and we would love to hear your thoughts on any of the subjects we discuss in this series. Please feel free to make your voice heard as part of the OpenSearch community through our [community meetings](https://www.meetup.com/OpenSearch/), [forum discussions](https://forum.opensearch.org/), and [GitHub repository](https://github.com/opensearch-project).
+
+## OpenSearch is for everyone, and we are inclusive
+
+We’d like to begin this ongoing conversation with a subject that is foundational to who we are and how we talk to each other as a community: inclusion. As an open-source project, OpenSearch is for everyone, and we are inclusive. We value the diversity of backgrounds and perspectives in the OpenSearch community and welcome feedback from any contributor, regardless of their experience level. 
+
+When developing OpenSearch documentation, we strive to create content that is inclusive and free of bias. We use inclusive language to connect with the diverse and global OpenSearch audience, and we are careful in our word choices. Inclusive and bias-free content improves clarity and accessibility of our content for all audiences, so we avoid ableist and sexist language and language that perpetuates racist structures or stereotypes. In practical terms, this means that we do not allow certain terms to appear in our content, and we avoid using others, depending on the context.
+
+## Inclusive terminology
+
+The following tables list offensive terms that we do not use in our content and sensitive terms that we avoid using. 
+
+We do not use the following terms in OpenSearch content. Instead, use one of the acceptable alternatives provided in the second column.
+
```

Comment:
Minor edit - please insert this above the table:

`<div class="table-styler"></div>`

thanks